docs/index.asciidoc

```diff
@@ -1185,7 +1185,7 @@ How long to wait before checking for a stale connection to determine if a keepal
 Consider setting this value lower than the default, possibly to 0, if you get connection errors regularly.
 
 This client is based on Apache Commons. Here's how the
-https://hc.apache.org/httpcomponents-client-ga/httpclient/apidocs/org/apache/http/impl/conn/PoolingHttpClientConnectionManager.html#setValidateAfterInactivity(int)[Apache
+https://hc.apache.org/httpcomponents-client-5.2.x/current/httpclient5/apidocs/org/apache/hc/client5/http/impl/io/PoolingHttpClientConnectionManager.html#setValidateAfterInactivity-org.apache.hc.core5.util.TimeValue-[Apache
```

Comment:
I had the [PR for same fix](https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/1101/files) (feel free to ignore mine) but I wonder if it is `5.x` versions. When I pulled `11.12.1` plugin version, it showed me `4.x`

Reply:
You are right,  
Manticore `[0.8.0-0.9.1]` comes with Http Client `4.5` https://github.com/cheald/manticore/blob/8d5e4768bf898fc2d87cf8ee7bbb585bce0693ec/manticore.gemspec#L36

So I close this in favor of your. But your also updates `.ci/logstash-run.sh` is that intended or part of the change?

Reply:
`.ci/logstash-run.sh` change was also intentional since CI was failing due to ruby bundle exemption (we did before).